src/prototype_room_external.js

```diff
@@ -112,6 +112,8 @@ Room.prototype.externalHandleHighwayRoom = function() {
     return;
   }
 
+  Memory.powerBanks = Memory.powerBanks || {};
```

Comment:
Move it to the top of the function and you can even remove the `if (Memory.powerBanks)`